cob-pcs/cob.bash.bashrc.t

```diff
@@ -1,2 +1,6 @@
+function parse_git_branch () {
+  git branch 2> /dev/null | sed -e '/^[^*]/d' -e 's/* \(.*\)/ (\1)/'
+}
+
 # set color fo all users
-PS1='${debian_chroot:+($debian_chroot)}\[\033[01;34m\]\u@\h\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\]\$ '
+PS1='${debian_chroot:+($debian_chroot)}\[\033[01;34m\]\u@\h\[\033[00m\]:\[\033[01;34m\]\w\[\033[0;33m\]$(parse_git_branch)\[\033[0m\]\$ '
```

Comment:
What's the difference between the `cob.bash.bashrc.X` files?
Can't they use a `cob.bash.bashrc`? Or at least symlinks?